dist/index.js

```diff
@@ -1062,11 +1088,19 @@ function loadConfig() {
         if (!fs_1.default.existsSync(filename)) {
             core.setFailed(`Could not find pipeline at ${baseFolder}/${pipeline}`);
         }
+        core.info(`file parameters ${runtimeParametersFile}`);
+        core.info(`folder parameters ${folderName}`);
+        if (runtimeParametersFile) {
+            core.info("Reading runtime parameters file");
```

Comment:
This other message I think you can remove it, or move the consolidated one into here. Otherwise it is going to be too noisy even for debug.

Reply:
I added that line for debugging I already deleted